packages/stylelint-plugin-ckeditor5-rules/tests/rules/license-header.test.js

```diff
@@ -0,0 +1,298 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { getTestRule } = require( 'jest-preset-stylelint' );
+const path = require( 'path' );
+const util = require( 'util' );
+
+const PLUGIN_PATH = path.join( __dirname, '..', '..', 'lib', 'rules', 'license-header.js' ).replace( /\\/g, '/' );
+
+global.testRule = getTestRule();
+
+const config = [
+	true,
+	{
+		headerContent:
+			[
+				' * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.',
+				' * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license'
+			]
+	}
+];
+
+const { ruleName } = require( PLUGIN_PATH );
+const messages = {
+	missing: `This file does not begin with a license header. (${ ruleName })`,
+	notLicense: `This file begins with a comment that is not a license header. (${ ruleName })`,
+	content: `Incorrect license header content. (${ ruleName })`,
+	gap: `Disallowed gap before the license. (${ ruleName })`
+};
+
+// For reasons that we don't understand, the `jest-preset-stylelint` package created additional `describe()` blocks for
+// the plugin configuration and the checked code. It uses the `util.inspect()` function for making a string from the given `input`.
+// Lets override it and return an empty string for these values to avoid a mess in a console.
+// The original function is restored at the end of the file.
+const defaultInspectFunction = util.inspect;
+
+util.inspect = () => {
+	return '';
+};
```

Comment:
I'd revert the line to the version I proposed in the past. If any code located between these tests and their execution wants to use the `util.inspect()` function, we overridden its behavior and perhaps, break the logic in unknown places.

```suggestion
util.inspect = function( input ) {
	// To hide: https://github.com/stylelint/jest-preset-stylelint/blob/3606955a27a22be789b9372b5dafaaab25401f7f/getTestRule.js#L172.
	if ( input === config ) {
		return '';
	}

	// To hide: https://github.com/stylelint/jest-preset-stylelint/blob/3606955a27a22be789b9372b5dafaaab25401f7f/getTestRule.js#L173.
	if ( input.startsWith && input.startsWith( '/*' ) ) {
		return '';
	}

	return inspect( input );
};
```